dbms/src/Functions/CMakeLists.txt

```diff
@@ -3,6 +3,12 @@ include(${ClickHouse_SOURCE_DIR}/cmake/dbms_glob_sources.cmake)
 add_headers_and_sources(clickhouse_functions ./GatherUtils)
 add_headers_and_sources(clickhouse_functions .)
 
+add_custom_command(
+	OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/tldLookup.cpp
+	COMMAND ${GPERF} ${CMAKE_CURRENT_SOURCE_DIR}/gperf/tldLookup.gperf --output-file=${CMAKE_CURRENT_SOURCE_DIR}/tldLookup.cpp
```

Comment:
Why use CMAKE_CURRENT_SOURCE_DIR?
Much better is CMAKE_CURRENT_BINARY_DIR for auto-generated files.